lists/hk.csv

```diff
@@ -39,3 +39,15 @@ https://www.vxcon.hk/,HACK,Hacking Tools,2019-07-10,OONI partner,
 https://unwire.hk/,CULTR,Culture,2019-07-10,OONI partner,
 https://hall1c.com/,CULTR,Culture,2019-07-10,OONI partner,
 https://kengowrites.blogspot.com/,POLR,Political Criticism,2019-07-10,OONI partner,
+https://www.hongkongfp.com/,NEWS,News Media,2019-08-30,OONI partner,
+https://www.demosisto.hk/,POLR,Political Criticism,2019-08-30,OONI partner,
+https://www.expressvpn.com/,ANON,Anonymization and circumvention tools,2019-08-30,OONI partner,
+https://telegram.org/,COMT,Communication Tools,2019-08-30,OONI partner,
+https://t.me/,COMT,Communication Tools,2019-08-30,OONI partner,
+https://telegra.ph/,HOST,Hosting and Blogging Platforms,2019-08-30,OONI partner,
+https://lih.kg/,POLR,Political Criticism,2019-08-30,OONI partner,
+https://matters.news/,HOST,Hosting and Blogging Platforms,2019-08-30,OONI partner,
+https://www.whatsapp.com/,COMT,Communication Tools,2019-08-30,OONI partner,
```

Comment:
We already test this inside of the global list so it should not be added to the country list.